pkg/destroy/vsphere/vsphere.go

```diff
@@ -217,12 +217,14 @@ func (o *ClusterUninstaller) Run() error {
 		o.Logger.Errorln(err)
 		return err
 	}
+	o.Logger.WithField("Tag", o.InfraID).Info("Destroyed")
```

Comment:
For consistency, if we are going to add the field to the "Destroyed" log event, we should add it to the other log events for the action. Specifically, use the logger returned by the call to `WithField` for the "Delete tag" log event and the error log event as well.

Reply:
Ok. I think the new changes are in line with this suggestion.